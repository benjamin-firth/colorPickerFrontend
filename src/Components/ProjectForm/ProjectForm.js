import React from 'react';
import './ProjectForm.scss';
import PropTypes from 'prop-types';

function ProjectForm() {
  return (
    <form className='project-form'>
      <p>Create New Project:</p>
      <input type='text' />
      <button className='submit-project-button'>Save Project</button>
    </form>
  );
}

export default ProjectForm;
