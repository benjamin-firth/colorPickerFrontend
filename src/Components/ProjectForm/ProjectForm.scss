h4 {
  font-size: 1.8vw;
  margin: 0;
  margin-top: 1.5vw;
}

.submit-project-button {
  box-shadow: 3px 3px;
  background-color: #CDCDCD;
}
