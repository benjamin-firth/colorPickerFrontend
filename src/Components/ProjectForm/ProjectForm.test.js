import React from 'react';
import { render } from '@testing-library/react';
import ProjectForm from './ProjectForm';
import { shallow } from 'enzyme';

describe('', () => {
  it('should', () => {

  })
});
