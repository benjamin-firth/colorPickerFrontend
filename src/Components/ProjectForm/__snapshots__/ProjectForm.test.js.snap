// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ProjectForm should match the snapshot 1`] = `
<form
  className="project-form"
  onSubmit={[Function]}
>
  <h4>
    Create New Project:
  </h4>
  <input
    onChange={[Function]}
    type="text"
  />
  <button
    className="submit-project-button"
    type="submit"
  >
    Save Project
  </button>
</form>
`;
